onboarding-manager/pkg/platformbundle/ubuntu-22.04/Installer

```diff
@@ -280,6 +280,10 @@ install_other_agents() {
         inbc-program="$INBC_PROGRAM_VERSION" \
         platform-update-agent="$PLATFORM_UPDATE_AGENT_VERSION" \
         platform-telemetry-agent="$PLATFORM_TELEMETRY_AGENT_VERSION"
+
+      echo "Update INBM configuration file"
+      # Remove docker checks from INBM configuration file
+      sed -i '/<sotaSW>/,/</sotaSW>/s/docker//' /etc/intel_manageability.conf
```

Comment:
Only removing docker sufficient or we need to remove trtl and inbm-telemetry as well?

Reply:
We can remove trtl and inbm-telemetry since there are no Docker-related actions required for INBM.

Reply:
Script updated.